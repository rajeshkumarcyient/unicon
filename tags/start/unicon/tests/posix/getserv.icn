
$include "posix.icn"

procedure main()

   s := getserv("whois") | stop("Couldn't get service \"finger\"")
   sdump(s)

end

procedure sdump(s)
   writes(s.name, "\t\t", s.port, "/", s.proto)
   writes("\t\t#", s.aliases)
   write()
end
