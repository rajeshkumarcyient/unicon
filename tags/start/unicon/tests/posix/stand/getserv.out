whois		43/tcp		#nicname
